spec/key_spec.rb

```diff
@@ -1,4 +1,4 @@
-RSpec.describe Epics::Key do
+RSpec.describe Epics::SignatureAlgorithm::RsaPss do
```

Comment:
It might make sense to change the filename and location accordingly to the class it tests